dev-tools/clean-setup.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+cd "$(dirname "$0")" || exit 1
```

Comment:
please add the header as shown here: https://github.com/e-m-b-a/embark/blob/20d2638f1f09855c3722c473660fb490b80d5ab7/dev-tools/check_project.sh#L1